.ci/validation.sh

```diff
@@ -428,6 +428,11 @@ spotbugs-and-pmd)
   exit "$RESULT"
 ;;
 
+rewrite)
+  export MAVEN_OPTS='-Xmx4g'
+  ./mvnw -e --no-transfer-progress clean rewrite:dryRun
```

Comment:
we can simply run rewrite? not a dry run
and there should be already post main command shell script that check that no diff in left in git repository